h/views/admin/users.py

```diff
@@ -37,8 +37,7 @@ def users_index(request):
 
     if user is not None:
         svc = request.find_service(name='annotation_stats')
-        counts = svc.user_annotation_counts(user.userid)
-        user_meta['annotations_count'] = counts['total']
+        user_meta['annotations_count'] = svc.user_annotation_count(user.userid)
```

Comment:
The query I mentioned above though can be satisfied using a simple index-only scan:

```
postgres=# explain select count(*) from annotation where userid = 'foo';
                                            QUERY PLAN                                             
---------------------------------------------------------------------------------------------------
 Aggregate  (cost=8.16..8.17 rows=1 width=0)
   ->  Index Only Scan using ix__annotation_userid on annotation  (cost=0.14..8.16 rows=1 width=0)
         Index Cond: (userid = 'foo'::text)
(3 rows)

postgres=# 
```

I anticipate this would be even faster than querying Elasticsearch.

Reply:
In the case where there aren't a lot of annotations for a user, the query is fast but that's not the case I'm trying to fix here, I'm trying to fix the case where there are a tone of annotations and the query is extremely slow.
![image](https://user-images.githubusercontent.com/30059933/46622239-aaafd280-cade-11e8-8894-c5aef8ea3c5f.png)
![image](https://user-images.githubusercontent.com/30059933/46622826-a5538780-cae0-11e8-8d32-9a6f60385df3.png)

Here's an example query from scibot:
![image](https://user-images.githubusercontent.com/30059933/46622314-fb273000-cade-11e8-90e5-a7b82bf8350a.png)
![image](https://user-images.githubusercontent.com/30059933/46622497-b0f27e80-cadf-11e8-8e7e-76451aab9602.png)

I think it's also worth considering we won't be running this query at all in most cases if we make this change thus it will always be faster. (I haven't seen a slow query that has other search criteria in it so generally I think it's fair to say this query will almost always be skipped.)